src/component/organism/menu/ProfileMenu.js

```diff
@@ -4,8 +4,7 @@ import {FlatList, Text, TouchableOpacity, View, ScrollView} from 'react-native';
 import {GRAY10} from 'Root/config/color';
 import {txt} from 'Root/config/textstyle';
 import {FavoriteTag46_Filled, FavoriteTag48_Filled, NextMark, Paw46} from 'Atom/icon';
-import {profileMenu} from 'Organism/style_organism';
-
+import {profileMenu} from '../style_organism copy';
```

Comment:
import 시에 상대참조보다 절대참조로 가도록 변경 'Organism/style_organism copy'로 변경해야함
```suggestion
import {profileMenu} from 'Organism/style_organism copy';
```